config/deployments/control-plane/cass-operator-dev/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: k8ssandra-operator
+
+resources:
+  - ../../default
+  - ../../../../../cass-operator/config/deployments/default
+
+images:
```

Comment:
Is this actually needed? The image is set to newTag: latest in the cass-operator in any case. Unless you actually point to a branch in your local repo, but in that case you'll run into issues since latest does not necessarily match other configs..

Reply:
Can be addressed later.